fee/src/lib.rs

```diff
@@ -1,16 +1,14 @@
 use {solana_sdk::fee::FeeDetails, solana_svm_transaction::svm_message::SVMMessage};
 
-/// Calculate fee for `SanitizedMessage`
+/// Calculate fee for `SVMMessage`
```

Comment:
nothing to do with this PR, i just noticed this comment was not updated previously